meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/salvator-generic-doma-virtio.dts

```diff
@@ -0,0 +1,29 @@
+/dts-v1/;
+
+#include <dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+
+	#address-cells = <2>;
+	#size-cells = <2>;
+
+	passthrough {
+		compatible = "simple-bus";
+		ranges;
+		#address-cells = <2>;
+		#size-cells = <2>;
+	};
+
+	firmware {
+		android {
+			compatible = "android,firmware";
+
+			vbmeta {
+                    compatible = "android,vbmeta";
```

Comment:
Something funny with the formatting

Reply:
Hi @lorc, accepted. This *.dts was using tabs, while I've used spaces.

Reply:
Hi @lorc, done.